src/idpf.rs

```diff
@@ -1662,6 +1662,7 @@ mod tests {
         }
     }
 
```

Comment:
@divergentdave Updating this would take some time. I'm happy to if you think this test is useful, but I'm wondering if it's appropriate to remove it. Consider that we'll have to update the hard-coded values every time we rev VERSION.